client/questions/answerPhotoDisplay.jsx

```diff
@@ -0,0 +1,30 @@
+import { render } from 'enzyme';
+import React from 'react';
+
+class AnswerPhotoDisplay extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      photos: this.props.photos
```

Comment:
I think using props in the state here will cause issues when the user switches products